src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-sonarjs/no-duplicate-string.ts

```diff
@@ -0,0 +1,15 @@
+import { RuleConverter } from "../../ruleConverter";
+
+export const convertNoDuplicateString: RuleConverter = (tslintRule) => {
+    return {
+        rules: [
+            {
+                ...(tslintRule.ruleArguments.length !== 0 && {
+                    ruleArguments: tslintRule.ruleArguments,
+                }),
```

Comment:
I really appreciate you adhering to the existing repository conventions, by the way 🥰 . This little one is a good example.